# Draw Pantera's panther as the product icon in WebUI pages.
#
# cr:chrome-product, Chromium's logo in the WebUI icon set, marks "About
# Pantera" in Settings' menu and other product entries. This replaces both of
# its definitions (24 and 960 units) with the panther glyph (DECISION-10),
# generated by scripts/make-icon.swift as branding/panther-glyph.svg. The
# panther's eye is cut out with the even-odd fill rule. See devbacklog TASK-19.

--- a/ui/webui/resources/cr_elements/icons.html.ts
+++ b/ui/webui/resources/cr_elements/icons.html.ts
@@ -95,7 +95,7 @@
         <path d="m13.05 12-4.1-4.102a.87.87 0 0 1-.274-.636.87.87 0 0 1 .273-.637.879.879 0 0 1 1.277 0l4.75 4.75a.84.84 0 0 1 .188.29c.04.105.063.218.063.335 0 .117-.024.23-.063.336a.84.84 0 0 1-.187.289l-4.75 4.75a.822.822 0 0 1-.625.262.902.902 0 0 1-.625-.285.883.883 0 0 1-.278-.641.88.88 0 0 1 .278-.637Zm0 0"></path>
       </g>
       <g id="chrome-product">
-        <path d="M9.45 14.574c.698.7 1.55 1.051 2.55 1.051 1 0 1.852-.352 2.55-1.05.7-.7 1.052-1.552 1.052-2.552s-.352-1.847-1.051-2.546C13.85 8.773 13 8.426 12 8.426c-1 0-1.852.347-2.55 1.05-.7.7-1.052 1.547-1.052 2.547 0 1 .352 1.852 1.051 2.551ZM12 17.426c.184 0 .371-.004.563-.012.19-.012.378-.039.562-.09l-2.45 4.2c-2.402-.266-4.378-1.31-5.937-3.126-1.558-1.816-2.34-3.941-2.34-6.375 0-.714.082-1.418.239-2.109.16-.691.398-1.371.715-2.039l4 6.852A4.988 4.988 0 0 0 9.3 16.71a5.346 5.346 0 0 0 2.699.715Zm0-10.801a5.114 5.114 0 0 0-3.164 1.05 5.499 5.499 0 0 0-1.938 2.7L4.45 6.125a8.93 8.93 0 0 1 3.313-2.738A9.61 9.61 0 0 1 12 2.426c1.582 0 3.074.375 4.477 1.125a9.739 9.739 0 0 1 3.449 3.074Zm8.898 1.8c.25.583.43 1.177.54 1.774.109.602.164 1.211.164 1.824 0 2.586-.836 4.754-2.5 6.516-1.668 1.758-3.793 2.777-6.375 3.063l3.921-6.875c.235-.418.418-.852.551-1.301a5.058 5.058 0 0 0-.176-3.324A5.943 5.943 0 0 0 16 8.426Zm0 0"></path>
+        <path fill-rule="evenodd" d="M21.81 21.52C23.06 21.26 23.23 19.75 22.33 19.02C21.42 16.08 20.34 12.42 19.05 10.26C18.19 8.97 17.11 7.89 16.03 7.46C15.86 6.81 15.69 5.74 15.51 5.26C15.26 5.43 14.09 6.17 13.62 6.64C12.15 5.95 10.64 4.66 9.89 4.23C9.35 2.93 8.23 1.34 7.71 0.48C7.19 0.78 5.99 2.29 5.47 3.58C4.95 4.66 4.30 6.17 4.26 7.68C3.52 10.28 1.68 15.58 0.00 23.26L0.00 24.00L20.75 24.00C21.30 23.26 21.75 22.34 21.81 21.52ZM14.22 12.04C15.52 11.90 16.70 12.85 16.64 14.37C14.93 14.68 13.99 13.49 14.22 12.04Z"></path>
       </g>
       <g id="close">
         <path d="m12 13.273-4.102 4.102a.816.816 0 0 1-.625.262.91.91 0 0 1-.625-.285.879.879 0 0 1 0-1.278L10.727 12 6.625 7.898a.83.83 0 0 1-.262-.636.914.914 0 0 1 .285-.637.879.879 0 0 1 1.278 0L12 10.727l4.102-4.102a.87.87 0 0 1 .636-.273.87.87 0 0 1 .637.273.87.87 0 0 1 .273.637.87.87 0 0 1-.273.636L13.273 12l4.102 4.102c.184.183.273.39.273.625 0 .23-.09.441-.273.625a.87.87 0 0 1-.637.273.87.87 0 0 1-.636-.273Zm0 0"></path>
@@ -318,7 +318,7 @@
         <path d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"></path>
       </g>
       <g id="chrome-product" viewBox="0 -960 960 960">
-        <path d="M336-479q0 60 42 102t102 42q60 0 102-42t42-102q0-60-42-102t-102-42q-60 0-102 42t-42 102Zm144 216q11 0 22.5-.5T525-267L427-99q-144-16-237.5-125T96-479q0-43 9.5-84.5T134-645l160 274q28 51 78 79.5T480-263Zm0-432q-71 0-126.5 42T276-545l-98-170q53-71 132.5-109.5T480-863q95 0 179 45t138 123H480Zm356 72q15 35 21.5 71t6.5 73q0 155-100 260.5T509-96l157-275q14-25 22-52t8-56q0-40-15-77t-41-67h196Z">
+        <path fill-rule="evenodd" d="M872.4 -99.2C922.4 -109.6 929.2 -170 893.2 -199.2C856.8 -316.8 813.6 -463.2 762 -549.6C727.6 -601.2 684.4 -644.4 641.2 -661.6C634.4 -687.6 627.6 -730.4 620.4 -749.6C610.4 -742.8 563.6 -713.2 544.8 -694.4C486 -722 425.6 -773.6 395.6 -790.8C374 -842.8 329.2 -906.4 308.4 -940.8C287.6 -928.8 239.6 -868.4 218.8 -816.8C198 -773.6 172 -713.2 170.4 -652.8C140.8 -548.8 67.2 -336.8 0 -29.6L0 0L830 0C852 -29.6 870 -66.4 872.4 -99.2ZM568.8 -478.4C620.8 -484 668 -446 665.6 -385.2C597.2 -372.8 559.6 -420.4 568.8 -478.4Z">
         </path>
       </g>
       <g id="close">
